src/ActiveLogin.Authentication.BankId.Api/Models/IAuthRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ActiveLogin.Authentication.BankId.Api.Models
+{
+    public interface IAuthRequest
```

Comment:
I know that Auth and Sign share these properties, but would like to keep the requests as clean as possible, every public interface / class we add needs to be supported.

I don't think that the interface adds any value at the moment, except for the simulated client. I'd vote for removing the interface and handle the scenario manually.